x-pack/plugins/dashboard_enhanced/public/components/dashboard_drilldown_config/dashboard_drilldown_config.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiFormRow, EuiSelect, EuiSwitch } from '@elastic/eui';
+import { txtChooseDestinationDashboard } from './i18n';
+
+export interface DashboardItem {
+  id: string;
+  title: string;
+}
+
+export interface DashboardDrilldownConfigProps {
+  activeDashboardId?: string;
+  dashboards: DashboardItem[];
+  currentFilters?: boolean;
+  keepRange?: boolean;
+  onDashboardSelect: (dashboardId: string) => void;
+  onCurrentFiltersToggle?: () => void;
+  onKeepRangeToggle?: () => void;
+}
+
+export const DashboardDrilldownConfig: React.FC<DashboardDrilldownConfigProps> = ({
+  activeDashboardId,
+  dashboards,
+  currentFilters,
+  keepRange,
+  onDashboardSelect,
+  onCurrentFiltersToggle,
+  onKeepRangeToggle,
+}) => {
+  // TODO: use i18n below.
```

Comment:
Can we do this TODO as part of this PR or is this in @mattkime s?

Reply:
It is in Matt's PR: https://github.com/elastic/kibana/pull/60087/files#diff-c7e77073659d2b600f5e6a59fcec15b0R58-R71